composer.json

```diff
@@ -23,6 +23,6 @@
 	},
 	"scripts": {
 		"format": "phpcbf --standard=phpcs.xml.dist --report-summary --report-source",
-		"lint": "phpcs --standard=phpcs.xml.dist"
+		"lint": "phpcs --standard=phpcs.xml.dist --runtime-set ignore_warnings_on_exit 1"
```

Comment:
Why this change? Seems unrelated?

Reply:
Without that change the lint warnings would cause the test to fail on 7 warnings for

`VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable`

```
Script phpcs --standard=phpcs.xml.dist handling the lint event returned with error code 1
✖ Command failed with exit code 1
```